lib/hooks/index.js

```diff
@@ -114,6 +118,16 @@ function activate(agent) {
     moduleHook.patch(instrumentation.module, agent, instrumentation.version);
   }
 
+  for (var hookName in toInstrument) {
+    var regex = new RegExp(path.sep + hookName + path.sep);
+    for (var file in require.cache) {
+      if (file.match(regex)) {
+        logger.warn('Module ' + hookName + ' was loaded before trace agent.');
```

Comment:
`node_modules` check added. File is now printed in addition to hook name for more information.